Fix creating simulator in Xcode 15

ForeverQ opened the pull request:
This PR removes quotes on "device_type" and "runtime" parameters to fix a simulator-not-found error in Xcode 15 beta. Tested on Xcode 15 and Xcode 14.

Commit:
Fix creating simulator in Xcode 15

GyroJoe commented:
Poetry install needs to be updated to:
```
curl -sSL https://install.python-poetry.org | python3 -
```
It looks like
https://python-poetry.org/docs/#installing-with-the-official-installer

dalemyers commented:
I've done some modernisation. Can you rebase and see if the build is happier?

ForeverQ commented:
Sure. I've rebased and updated.

It seems the bin location of Poetry needs to be updated to `/Users/runner/.local/bin`
https://dalemyers.visualstudio.com/Github/_build/results?buildId=288&view=logs&j=55d3718a-3d75-56f3-9d71-5698bcfe6174&t=745eb1bc-0de4-5a67-fed1-afcaf97849a1

dalemyers commented:
Published as version 15.0.0
